app/javascript/packages/form-steps/form-steps.spec.tsx

```diff
@@ -170,6 +175,18 @@ describe('FormSteps', () => {
     expect(onStepChange.calledOnce).to.be.true();
   });
 
+  it('calls trackEvent when personal key step is displayed', () => {
+    render(<FormSteps steps={[{ name: 'personal_key', form: () => <>N/A</> }]} />);
+
+    expect(analytics.trackEvent).to.have.been.calledWith('IdV: personal key visited');
+  });
+
+  it('calls trackEvent when personal key confirm is displayed', () => {
+    render(<FormSteps steps={[{ name: 'personal_key_confirm', form: () => <>N/A</> }]} />);
```

Comment:
If we don't care to be rendering anything of substance, we could return `null`:

```suggestion
    render(<FormSteps steps={[{ name: 'personal_key', form: () => null }]} />);

    expect(analytics.trackEvent).to.have.been.calledWith('IdV: personal key visited');
  });

  it('calls trackEvent when personal key confirm is displayed', () => {
    render(<FormSteps steps={[{ name: 'personal_key_confirm', form: () => null }]} />);
```